docs/docs/configuration.md

```diff
@@ -6,15 +6,15 @@ For most deployments, we recommend to set at least the following variables: `LAK
 
 ### General
 
-| Variable                                         | Example                                | Description |
-|--------------------------------------------------|----------------------------------------|-----|
-| <nobr>`LAKEKEEPER__BASE_URI`</nobr>              | <nobr>`https://example.com:8181`<nobr> | Base URL where the catalog is externally reachable. Default: `https://localhost:8181` |
+| Variable                                         | Example                                | Description                                                                                                                                                                                                                                                               |
+|--------------------------------------------------|----------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| <nobr>`LAKEKEEPER__BASE_URI`</nobr>              | <nobr>`https://example.com:8181`<nobr> | Fallback base URL where the catalog is externally reachable. Default: `https://localhost:8181`. Lakekeeper uses x-forwarded-for & host before falling back to this option.                                                                                                |
```

Comment:
Let's name all 3 variables that we respect explicitly. Helps for search :).

Lets also describe what we do if they are not set.